### PR TITLE
update admonition styles in asciidoc content

### PR DESCRIPTION
**Pull request by rosieyohannan**
Update asciidoc admonitions with to look like html alerts.

This is how they looked before:
Screenshot 2022-04-05 at 14 29 05

This is how they look now:

Screenshot 2022-04-06 at 23 30 42

I have updated the style guide section on formatting too, to include info on how to use them.

### CODE REVIEW

File: jekyll/_style/style-formatting.adoc [hunk 1]
```diff
@@ -11,5 +11,88 @@
...
+NOTE: If the paragraph text is not styled using `[.style]##` as shown here, it will still work but the text colour will be black rather that matching the colour of the admonition.
+
+=== Things to avoid
+
+Try not to use links in warnings at this time due to the blue on red style:
+
+[source.adoc]
+WARNING: A warning with a link:circleci.com[CircleCI] is hard to read.
+
+WARNING: A warning with a link:circleci.com[CircleCI] is hard to read.
```
Comment (rosieyohannan) [inline jekyll/_style/style-formatting.adoc:98]:
```suggestion
WARNING: A warning with a link:https://circleci.com/[CircleCI] is hard to read.

WARNING: A warning with a link:https://circleci.com/[CircleCI] is hard to read.
```

**Review by catherinepope — APPROVED** [review]
👍

**Review by tshen815 — APPROVED** [review]
Looks great!

**Comment by rsylvian** [comment]
Much better

**Pull request merged**